mod/01_rating/set/type/metric.rb

```diff
@@ -16,6 +17,64 @@
   def view_caching?
     true
   end
+
+  view :value_type_edit_modal_link do |args|
```

Comment:
this should probably be in right/value_type  (or type_plus_right/metric/value_type, but the first one is probably fine)